app/utils/auth.py

```
@@ -1,134 +1,173 @@
+
+        if not auth_result.is_signed_in:
+            reason = auth_result.reason.name if auth_result.reason else "UNKNOWN"
+            message = auth_result.message or "Authentication failed for unknown reasons."
+
+            raise UnauthorizedAccess(
+                log_message=f"Clerk failed to authenticate | Reason: {reason} | Message: {message}"
+            )
+
+        payload = auth_result.payload
+        user = UserClaims(**payload)
+        user_id_var.set(user.sub)
```

Ruff (0.11.9)

Undefined name `user_id_var`

(F821)